src/platform/Linux/BLEManagerImpl.cpp

```diff
@@ -204,12 +204,12 @@ CHIP_ERROR BLEManagerImpl::ConfigureBle(uint32_t aAdapterId, bool aIsCentral)
 
 CHIP_ERROR BLEManagerImpl::StartBLEAdvertising()
 {
-    return StartBluezAdv(mpEndpoint);
+    return mEndpoint->StartAdvertisement();
```

Comment:
should we return an error if mEndpoint is null (like illegal state or something) ? 
here and below.

Reply:
I've just removed both these functions in favor of using `mEndpoint->StartAdvertisement()` directly in the `DriveBLEState` so it will be clear that `mEndpoint` is not possible to be null there.